### ISSUE TITLE
ERROR in multi ./src/styles.sas

### ISSUE THREAD

**Issue by developer239** [angular/angular-cli#13616]
# 🐞 Bug report

### Command (mark with an `x`)

```
- [ ] new
- [x] build
- [x] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?
I didn't use angular before.

### Description

Running `ng serve` ends up with error:
```
ERROR in multi ./src/styles.sass
Module not found: Error: Can't resolve '/Users/michaljarnot/IdeaProjects/ng7/src/styles.sass' in '/Users/michaljarnot/IdeaProjects/ng7'
```

![image](https://user-images.githubusercontent.com/14946081/52423493-87cb1480-2af8-11e9-8c84-950ab414ffae.png)

## 🔬 Minimal Reproduction

This is actually the first time I tried to run Angular application on this machine. On clean install Mac OS Mojave:

`npm install -g @angular/cli`
`ng new ng7`
`ng serve`

**Comment by developer239** [comment]
Editing `angular.json` fixed the problem:

![image](https://user-images.githubusercontent.com/14946081/52423886-4129ea00-2af9-11e9-9106-7ef4be3f5b96.png)

**Comment by wKoza** [comment]
duplicate #13550 and resolved by #13552
